.azure-pipelines/azure-pipelines-job-groups.yml

```diff
@@ -43,6 +43,7 @@ jobs:
       - ${{ parameters.preSteps }}
       - ${{ if ne(jobGroup.script, '') }}:
         - script: |
+            set -e
```

Comment:
>set -e [](start = 12, length = 6)

this is big change. Is it better to let caller decide in its own `jobGroup.script`?

Reply:
It is ok deleting this line in image build.